plugins.go

```diff
@@ -242,6 +242,7 @@ type JSVM struct {
 	VM      *otto.Otto
 	Timeout time.Duration
 	Log     *logrus.Logger // logger used by the JS code
+	RawLog  *logrus.Logger // logger used by the JS code
```

Comment:
these two comments probably need clarification now, i.e. that one is used in `log` and the other in `rawlog`